### PR TITLE
Feature/warn when replacing

### LINKED ISSUE #2115: Warn me sufficiently when I am replacing my wallet

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Adequately warn user when replacing wallet:

https://user-images.githubusercontent.com/675259/106075100-fbe29680-60da-11eb-8836-7ff74d8e9e35.mp4

**Checklist**

* [x] There is a related GitHub issue

**Issue**

Resolves #2115

### CODE REVIEW

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
- **Issue 1:** 
<strike>The wording seems a little bit incomplete where we say "setup new"

image

Maybe "and setup a new wallet" or "and setup a new one" 
cc: @cjeria @omnat </strike>

- **Issue 2:** 
<strike>Mispelling 

image

Should be "permanently"</strike>

- **Issue 3:** 
<strike>On a smaller screen the modal is stretched out (example iPhone 5s)

Screen Shot 2021-01-29 at 6 04 31 PM</strike>

- **Issue 4:** 
Delete confirmation doesn't pop up on iOS; seen here = http://recordit.co/A0w226up6u

- **Issue 5:** 
<strike>In the figma mock up, it states to re-use the 'Creating wallet' loading screen from onboarding flow, and then it says not to allow going back on the onboarding carousel from here.  Also I didn't see the notification that my wallet was erased.  All 3 are not implemented; seen here = http://recordit.co/DeEz3hGhIh</strike>

Screen Shot 2021-01-29 at 6 17 43 PM

**Comment by cjeria** [comment]
Issue 1: Let's go with "Can’t log in? You can ERASE your current wallet and **set up a new one**".

Issue 3: Let's try reducing the left and right margins of the modal so all sides have equal spacing (the area around the modal showing the background). Experiment with 8px, 16px or 24px to see what looks best.
image

File: app/components/Views/ManualBackupStep3/index.js [hunk 1]
```diff
@@ -70,14 +71,11 @@ const HARDWARE_BACK_PRESS = 'hardwareBackPress';
...
 class ManualBackupStep3 extends PureComponent {
-	static navigationOptions = ({ navigation }) => getOnboardingNavbarOptions(navigation);
+	static navigationOptions = ({ navigation }) => getTransparentOnboardingNavbarOptions(navigation);
 
 	constructor(props) {
 		super(props);
 		this.steps = props.navigation.getParam('steps', undefined);
-		props.navigation.setParams({
-			headerLeft: <View />
-		});
```
Comment (rickycodes) [inline app/components/Views/ManualBackupStep3/index.js:80]: this is just me undoing a previous silly thing i did.

File: app/components/Views/Onboarding/index.js [hunk 2]
```diff
@@ -550,6 +600,24 @@ class Onboarding extends PureComponent {
 		);
 	}
 
+	handleSimpleNotification = () => {
+		const notificationTitle = 'Wallet succesfully erased';
+		const notificationStatus = 'success';
```
Comment (estebanmino) [inline app/components/Views/Onboarding/index.js:605]: can you move this to locales?

Comment (rickycodes) [reply app/components/Views/Onboarding/index.js:605]: good catch

**Review by estebanmino — APPROVED** [review]
👌

**Review by estebanmino — APPROVED** [review]
lgtm

**Review by ibrahimtaveras00 — APPROVED** [review]
QA Passed 👍🏽

**Pull request merged**